kratos/linear_solvers/rayleigh_quotient_iteration_eigenvalue_solver.h

```diff
@@ -294,7 +294,7 @@ class RayleighQuotientIterationEigenvalueSolver : public IterativeSolver<TSparse
                     shift_value = ro;
                 }
 
-                noalias(shifted_k) = K - shift_value*M;
+                boost::numeric::ublas::noalias(shifted_k) = K - shift_value*M;

```

Comment:
ouch, not coming from now...but this operation is potentially very inefficient in ublas

Reply:
I see, nevertheless I'm not sure how we can improve it....

Reply:
@loumalouomega you have an optimized function to sum two matrices right?

Reply:
yes, I do it here or in a future PR?

Reply:
in a future PR, but write an issue to remind that...

Reply:
Adressed in #2832